components/org.wso2.carbon.identity.notification.push.common/pom.xml

```
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```

check other places as well
```suggestion
  ~ Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
```